packages/components/app/styles/components/table.scss

```diff
@@ -19,175 +19,167 @@ $hds-table-cell-padding-medium: 14px 16px 13px 16px; // the 1px difference is to
 $hds-table-cell-padding-short: 6px 16px 5px 16px; // the 1px difference is to account for the bottom border
 $hds-table-cell-padding-tall: 22px 16px 21px 16px; // the 1px difference is to account for the bottom border
 
+
+// TABLE
+
 .hds-table {
   width: 100%;
   border: $hds-table-border-width solid $hds-table-border-color;
   border-radius: $hds-table-border-radius;
   border-spacing: 0;
 }
 
-// TABLE DISPLAY
+
+// table display
+
 .hds-table--layout-fixed {
   table-layout: fixed;
 }
 
+
+// ----------------------------------------------------------------
+
 // TABLE HEADER
+
 .hds-table__thead {
   .hds-table__tr {
     color: var(--token-color-foreground-strong);
     background-color: var(--token-color-surface-strong);
-    // BORDER RADIUS: TARGET FIRST AND LAST TH ELEMENTS IN THE ROW
-    &:first-of-type {
-      th:first-child {
-        border-top-left-radius: $hds-table-inner-border-radius;
-      }
 
-      // LAST TH IN THE ROW, BORDER RADIUS
-      th:last-child {
-        border-top-right-radius: $hds-table-inner-border-radius;
+    th {
```

Comment:
Is there a reason for not using a class here (e.g `hds-table__td`)? I know this is not introduced in this PR, but we may want to consider relying on classes as much as possible.

Reply:
Trying to think if we're ever in a position where the class may not be applied 🤔

Reply:
that's how it was implemented initially, but that's a good point. I've refactored it in https://github.com/hashicorp/design-system/pull/1860/commits/c8a56788e14895fb05f9680a0e67bd59d5a2778d